docs/artifactory/dotnet/help.go

```diff
@@ -0,0 +1,8 @@
+package dotnet
+
+const Description = "Run .NET Core CLI"
+
+var Usage = []string{`jfrog rt dotnet <dotnet command> [command options]`}
```

Comment:
dotnet command --> dotnet sub-command

Let's also do the same change in the pip command help:
pip command --> pip sub-command